src/io/utils.cpp

```diff
@@ -167,7 +177,7 @@ namespace occa {
       expFilename = fo.expand(expFilename);
 
       if (makeAbsolute && !isAbsolutePath(expFilename)) {
-        expFilename = env::PWD + expFilename;
```

Comment:
Can we initialize `env::PWD` to be the result of `currentWorkingDirectory` rather than calling this function every time?

Reply:
Ok, did that now. I've also taken the liberty of renaming env::PWD to env::CWD to avoid possible confusion since it now doesn't reflect the actual ${PWD} variable anymore.